util/mkpasswd.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2015 Apcera Inc. All rights reserved.
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"crypto/rand"
+	"flag"
+	"fmt"
+	"log"
+	"math/big"
+
+	"golang.org/x/crypto/bcrypt"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+func usage() {
+	log.Fatalf("Usage: mkpasswd -p <stdin password> \n")
+}
+
+const (
+	PasswordLength = 22
+	Cost           = 11
```

Comment:
Magic number with security implications should have a comment describing how it was derived.